cmake/CPack.STGZ_Header.sh.in

```diff
@@ -141,8 +140,6 @@ if [ ! "$link_directory" = "n" ]; then
 		rm -f "${link_directory}/energyplus"
 		rm -f "${link_directory}/Energy+.idd"
 		rm -f "${link_directory}/Energy+.schema.epJSON"
-		rm -f "${link_directory}/EP-Compare"
-		rm -f "${link_directory}/EPMacro"
```

Comment:
Did you remove that EPMacro line by mistake?

Reply:
Thanks @jmarrec, the EPMacro line was a mistake and I will make the changes to verify_signature.py and getting started. 

The change to EP-Launch, I think I will skip for now. It fails gracefully, which I think is better than just making it disappear from the list of choices. But the next time I work on EP-Launch, I will pick it up for a future release. I added it to this [issue for the future](https://github.com/NREL/EnergyPlus/issues/10576)

Reply:
@jmarrec in verify_signature.py, shouldn't the list of the bundled apps include others like IDFVersionUpdater, ExpandObjects and EPMacro and other utilities that are available on MacOS? Or is it only for apps that had not included the source code in the repo?

Reply:
IDFVersionUpdater is in there. That BUNDLED_APPS is just so I don't recheck something I actually codesigned in the repo directly.

ExpandObjects is a built one (and a fortran exe, not a bundled app) so it'll be checked by default.

https://github.com/NREL/EnergyPlus/blob/22e9501303d86afcec015a1247d6ae32f4e7aeee/scripts/dev/verify_signature.py#L73-L77